### ISSUE TITLE
Bug: form-error overflows when error wraps to 2nd line

### ISSUE THREAD

**Issue by daniel-sc** [sbb-design-systems/sbb-angular#879]
**Describe the bug**
When the error message in a form-field is wrapping to a second (or even more) line(s), the error is overflowing to the following components - see screenshot:

![image](https://user-images.githubusercontent.com/117919/132496438-4498aa0f-5a6f-46f2-b116-ee9333d0f076.png)

**To Reproduce**
1. Go to Examples
2. Edit error to have more words
3. See error overflowing next component

**Expected behavior**
When the error wraps to multiple lines, the follwing components should be shifted/moved accordingly.

**Screenshots**
see above

**Desktop (please complete the following information):**
 - OS: win10
 - Browser Chrome 93.0
 - Version: SACL 12.3.0

**Additional context**
- maybe use `min-height` to allow form-field to expand if more space is needed?

**Comment by kyubisation** [comment]
See #853
Also see `Error messages` in https://angular.app.sbb.ch/business/components/form-field

**Comment by daniel-sc** [comment]
OK - so I might be convinced that according to your documentation this is not a bug, but rather a feature request?
Non the less, this "feature" probably prevents many bugs, as e.g. with translations the actual space needed is often not clear..

**Comment by kyubisation** [comment]
No, this should be solved. Add the css class `sbb-form-field-flexible-errors` to any ancester (e.g. `html`, `body`, etc.).

**Comment by daniel-sc** [comment]
The problem is that `sbb-form-field-flexible-errors`  removes the space for the first error line - which I'd like to retain.
So what we need (and I suppose, what most others want) is to have space reserved for the first error line, which is expanded in the (non standard) case that the error is larger.

**Comment by kyubisation** [comment]
Ah, I understand.
It is not necessarily what most others want, because we implemented it this way due to feedback we received.
This is not trivially possible, due to positioning and display issues.
